modules/mantisBidAdapter.js

```diff
@@ -240,6 +241,11 @@ const spec = {
       }),
       property: property
     };
+
+    if (bidderRequest && bidderRequest.gdprConsent && bidderRequest.gdprConsent.gdprApplies) {
+      query.consent = false;
```

Comment:
@parisholley what is the intent here, you don't want to collect consent string in GDPR zone (you're not part of the global vendor list)? You just mark this request as if consent was not given?

Reply:
correct, we don't have plans to track user data in the EU (most of our business focuses on the US), so just prefer to act as if we never have consent (which we do today on any inbound EU requests outside prebid).